site/get-started/common-steps/_access-quickstart-notebook.qmd

```diff
@@ -2,6 +2,6 @@
 
 You can either load the hosted version of our notebook, or download the notebook to run in your own local environment:
 
-#### Load the hosted notebook (recommended)
+#### Open the hosted notebook (recommended)
 
 1. Sign in to [JupyterHub]({{< var url.jupyterhub >}}) with your {{< var vm.product >}} login.
```

Comment:
This might be an indentation issue to tell Pandoc that the bullets should be numbered sequentially rather than "1." and "1.", please fix?

![Capto_ 2025-06-16_03-37-22_pm](https://github.com/user-attachments/assets/c91ae09b-7486-40ea-a432-40c6dcd2ac64)